packages/video_player/video_player/example/integration_test/video_player_test.dart

```diff
@@ -162,5 +162,24 @@ void main() {
       await tester.pumpAndSettle();
       expect(_controller.value.isPlaying, true);
     }, skip: kIsWeb); // Web does not support local assets.
+
+    testWidgets('test Android video is rotated for landscapeRight',
+        (WidgetTester tester) async {
+      _controller =
+          VideoPlayerController.asset('assets/keyboard-landscape.mp4');
+
+      VideoPlayer player = VideoPlayer(_controller);
+      await tester.pumpWidget(player);
+
+      expect(find.byType(Transform), findsOneWidget);
```

Comment:
@stuartmorgan is this a reasonable way to inspect the widget and determine if Transform.rotate is present?

Reply:
This isn't an area I have familiarity with; but it seems plausible 🤷🏻

Reply:
I ended up not going this route.